### PR TITLE
book: Use `Object::builder()` instead of `Object::new()`

### PR DESCRIPTION
**Pull request by Hofer-Julian**

### CODE REVIEW

File: book/listings/composite_templates/4/custom_button/mod.rs [hunk 1]
```diff
@@ -11,7 +11,7 @@ glib::wrapper! {
 
 impl CustomButton {
     pub fn new() -> Self {
-        Object::new(&[])
+        Object::builder().build()
```
Comment (bilelmoussaoui) [inline book/listings/composite_templates/4/custom_button/mod.rs:14]: I wonder if we can have Default implement for Object?

Comment (sdroege) [reply book/listings/composite_templates/4/custom_button/mod.rs:14]: You can't do a blanket implementation of a trait outside the crate. We could add a `default()` constructor though, or `impl Default for SpecificObject` via the `wrapper!` macro.

Comment (bilelmoussaoui) [reply book/listings/composite_templates/4/custom_button/mod.rs:14]: I meant in gtk-rs-core itself, but i suppose that would conflict with people having their own default implementation?

Comment (sdroege) [reply book/listings/composite_templates/4/custom_button/mod.rs:14]: We don't define the `Default` trait :)

**Pull request merged**